src/feed/Feed.js

```diff
@@ -135,7 +144,6 @@ class Feed {
                 });
                 result.quotes = TickHistoryFormatter.formatHistory(response);
             } catch (err) {
-                console.error(err);
```

Comment:
why did you remove this?

Reply:
@brucebinary I just think that is useless. I put it back.